app/routes/lending/components/LendingAdmin.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+  Button,
+  Card,
+  Flex,
+  Icon,
+  LoadingIndicator,
+} from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { useState } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { Link } from 'react-router-dom';
+import { fetchAllLendableObjects } from 'app/actions/LendableObjectActions';
+import { fetchAllLendingRequests } from 'app/actions/LendingRequestActions';
+import { Content } from 'app/components/Content';
+import NavigationTab from 'app/components/NavigationTab';
+import { selectAllLendableObjects } from 'app/reducers/lendableObjects';
+import { selectAllLendingRequests } from 'app/reducers/lendingRequests';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import { LendingRequestStatus } from 'app/store/models/LendingRequest';
+import styles from './LendingAdmin.css';
+import { RequestItem } from './RequestItem';
+
+const LendableObjectsAdmin = () => {
+  const [showOldRequests, setShowOldRequests] = useState(false);
+
+  const dispatch = useAppDispatch();
+
+  usePreparedEffect(
+    'fetchAllLendingObjectsAndRequests',
+    () => {
+      dispatch(fetchAllLendingRequests());
+      dispatch(fetchAllLendableObjects());
+    },
+    [],
+  );
```

Comment:
You have to return something. This will cause issues if you try to refresh

```suggestion
  usePreparedEffect(
    'fetchAllLendingObjectsAndRequests',
    () => (
      Promise.allSettled([
      	dispatch(fetchAllLendingRequests()),
     	dispatch(fetchAllLendableObjects()),
      )],
    ),
    [],
  );
```